Source/Renderer/AutomaticUniforms.js

```diff
@@ -1484,6 +1484,70 @@ define([
             getValue : function(uniformState) {
                 return uniformState.shadowMap.shadowMapMatrix;
             }
+        }),
+
+        /**
+         * An automatic GLSL uniform representing the sun's shadow map cascade far splits.
+         *
+         * @private
```

Comment:
I don't think `@private` is needed for any of these (including `czm_sunShadowMapTexture` and `czm_sunShadowMapMatrix`) since we are not generating doc for uniforms at all right now.